.teamcity/_self/builds/WPComPlugins_EditorToolkit.kt

```diff
@@ -124,6 +124,9 @@ object WPComPlugins_EditorToolKit : BuildType({
 				if ! diff -rq --exclude="*.asset.php" ./editing-toolkit-plugin/ ../../etk-release-build/ ; then
 					echo "The build is different from the last release build. Therefore, this can be tagged as a release build."
 					curl -X POST -H "Content-Type: text/plain" --data "etk-release-build" -u "%system.teamcity.auth.userId%:%system.teamcity.auth.password%" %teamcity.serverUrl%/httpAuth/app/rest/builds/id:%teamcity.build.id%/tags/
+					if [ `git symbolic-ref --short HEAD` == "trunk" ] ; then
```

Comment:
What do you think about using TeamCity's `%teamcity.build.branch.is_default%`? It probably won't happen again anytime soon, but avoid using `trunk` everywhere will help us with any future branch rename.

Reply:
Nice, I didn't know about that! Added.